msibi/state.py

```diff
@@ -33,9 +51,9 @@ class State(object):
         True if each query trajectory is backed up (default=False)
 
     """
-    def __init__(self, k, T, state_dir='', traj_file=None, top_file=None,
+    def __init__(self, kT, state_dir='', traj_file=None, top_file=None,
                  name=None, backup_trajectory=False):
-        self.kT = k * T
+        self.kT = kT
```

Comment:
I don't think so. HOOMD documentation states for v2.0.0
> Integrators now take kT arguments for temperature instead of T to avoid confusion on the units of temperature.

under the heading

> Changes that require job script modifications